Fixed linux scripts file permissions (they were lost by mistake)

From: Ziv Hendel <ziv@primesense.com>


---
 Platform/Linux/CreateRedist/RedistMaker |    0 
 Platform/Linux/CreateRedist/install.sh  |    0 
 2 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 Platform/Linux/CreateRedist/RedistMaker
 mode change 100644 => 100755 Platform/Linux/CreateRedist/install.sh

diff --git a/Platform/Linux/CreateRedist/RedistMaker b/Platform/Linux/CreateRedist/RedistMaker
old mode 100644
new mode 100755
diff --git a/Platform/Linux/CreateRedist/install.sh b/Platform/Linux/CreateRedist/install.sh
old mode 100644
new mode 100755
